### DISCUSSION TITLE
Export to PDF

### DISCUSSION THREAD

**Discussion by jaccurate-cmd** [chuangcaleb/obsidian-fountain-editor#30] [Ideas]
As title suggests, is there a way to export to a PDF? It would be very helpful to be able to share a properly formatted file to share with others who don't have this plugin/Obsidian.

**Comment by chuangcaleb** [comment]
There are many ways to export to PDF! Just not natively via this plugin.

You've got a local .fountain file! There's a bunch of really good compilers, just Google "fountain to pdf". I'm personally using [VS Code's Better Fountain Extension](https://marketplace.visualstudio.com/items?itemName=piersdeseilligny.betterfountain). You can set a whole bunch of settings. I think it itself is built on top of another library. 

I didn't want to reinvent the wheel when it came to features. Though I see how it can be useful to have a native option/button. For now, it's really low on priority. Let me know if you have further questions?
